tests/package.json

```diff
@@ -38,8 +38,8 @@
     "tracing-test-single": "ETHAPI_CMD='--ethapi=txpool,debug,trace' FORCE_WASM_EXECUTION='true' FORCE_COMPILED_WASM='true' WASM_RUNTIME_OVERRIDES='moonbase-overrides' mocha -r ts-node/register 'tracing-tests/test-trace-filter.ts'",
     "para-test": "mocha -r ts-node/register 'para-tests/**/test-*.ts'",
     "para-test-single": "mocha -r ts-node/register 'para-tests-no-ci/test-xcm-para.ts'",
-    "smoke-test": "mocha -r ts-node/register 'smoke-tests/**/test-*.ts'",
-    "smoke-test-single": "mocha -r ts-node/register 'smoke-tests/test-ethereum-current-consistency.ts'",
+    "smoke-test": "mocha -r ts-node/register --require ./util/smokeFixtures.ts 'smoke-tests/**/test-*.ts' --exit",
```

Comment:
please remove the `--exit`, it will silently leave if we don't cleanup properly.
We should add:
"mocha": "mocha -r ts-node/register --require ./util/smokeFixtures.ts"

Reply:
done